CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [6.4.0] - [TBD]
+### Added
+- Create `apiary_system` database and buckets. This is pre-fork for Ranger access logs Hive tables and other system data. Requires `apiary-metastore-docker` version `1.15.0` or above.
```

Comment:
```suggestion
- Create `apiary_system` database and buckets. This is pre-work for Ranger access logs Hive tables and other system data. Requires `apiary-metastore-docker` version `1.15.0` or above.
```
This is the 3rd time I have made this suggestion.  Please commit it so we don't have embarrassing typos in our open-source documentation.